### PR TITLE
Document EnableFileSearch Config Setting

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation task: https://mattermost.atlassian.net/browse/MM-36291

Updated:
- Self-Managed Admin Guide > Configure Mattermost > Configuration Settings
   - Added Service Settings > Enable File Search
   - Updated File Settings > Enable Document Search by Content to ensure that all 3 search configs are presented consistently
   - Updated File Settings > Enable Searching Content of Documents within ZIP Files to ensure that all 3 search configs are presented consistently

### CODE REVIEW

**Review by esethna — APPROVED** [review]

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -4788,6 +4798,19 @@ Data Prefetch
 | This feature's ``config.json`` setting is ``"ExperimentalDataPrefetch": true`` with options ``true`` and ``false``.                                                  |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable File Search
+^^^^^^^^^^^^^^^^^^
+
+This configuration setting enables users to search documents attached to messages by filename. To enable users to search documents by their content, you must also enable the ``ExtractContent`` configuration setting. See our `Enable Document Search by Content <https://docs.mattermost.com/administration/config-settings.html#enable-document-search-by-content>`__ documentation for details. Document content search is available in Mattermost Server from v5.35, with mobile support coming soon. 
+
+**True**: Supported document types are searchable by their filename.
```
Comment (jespino) [inline source/administration/config-settings.rst:4806]: The "supported" string is not needed here, actually, all files can be search by filename, the is not restriction in the format of the file itself.

**Review by jespino — APPROVED** [review]
I tiny detail to improve, otherwise looks good to me.

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -623,9 +623,9 @@ Maximum file size for message attachments entered in megabytes in the System Con
 Enable Document Search by Content
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Document content search is available in Mattermost Server from v5.35, with mobile support coming soon. Searching document contents adds load to your server. For large deployments, or teams that share many large, text-heavy documents, we recommended you review our `hardware requirements <https://docs.mattermost.com/install/requirements.html#hardware-requirements>`__, and test enabling this feature in a staging environment before enabling it in a production environment.
+This configuration setting enables users to search the contents of documents attached to messages.
 
-**True**: Supported document types are searchable by their content. Install `these dependencies <https://github.com/sajari/docconv#dependencies>`__ to extend content searching support to include DOCX, RTF, and PAGES files. 
+**True**: Supported document types are searchable by their content. Install `these dependencies <https://github.com/sajari/docconv#dependencies>`__ to extend content searching support to include additional file formats, including DOCX, RTF, and PAGES files.
```
Comment (cwarnermm) [inline source/administration/config-settings.rst:628]:
```suggestion
**True**: Document types are searchable by their content. Install `these dependencies <https://github.com/sajari/docconv#dependencies>`__ to extend content searching support to include additional file formats, including DOCX, RTF, and PAGES files. 
```

File: source/administration/config-settings.rst (hunk 1 above)
Comment (cwarnermm) [reply source/administration/config-settings.rst:4806]: Thanks!

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**